src/main/java/it/gov/pagopa/payhub/activities/activity/paymentsreporting/PaymentsReportingIngestionFlowFileActivityImpl.java

```diff
@@ -55,6 +59,7 @@ public PaymentsReportingIngestionFlowFileActivityResult processFile(Long ingesti
 			File ingestionFlowFile = retrieveFile(ingestionFlowFileDTO);
 
 			Pair<String, List<PaymentsReportingDTO>> pair = parseData(ingestionFlowFile, ingestionFlowFileDTO);
+			paymentsReportingInsertionService.savePaymentsReporting(pair.getRight());
```

Comment:
just call the DAO, let's avoid a Service layer ;)
Moreover we will not more have the need for another exception